hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsService.java

```diff
@@ -218,6 +218,15 @@ void resolveAlertsForTrigger(String tenantId, String triggerId, String resolvedB
     void sendData(Collection<Data> data) throws Exception;
 
     /**
+     * Send data into the alerting system for evaluation.
+     *
+     * @param data Not Null.  The data to be evaluated by the alerting engine.
+     * @param ignoreFiltering  An optimization. Set true *only* if you are sure the data is useful for evaluation.
+     * @throws Exception any problem.
+     */
+    void sendData(Collection<Data> data, boolean ignoreFiltering) throws Exception;
```

Comment:
A client shouldn't have visibility of this filtering process IMO.
Another thing is that for packages where we know that metrics or apm are in front and performing some kind of filtering process, then a flag at config should activate this.
But for API purposes I don't think this is appropiate.